src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptWellKnownSymbolTypes.cs

```diff
@@ -2,12 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Microsoft.CodeAnalysis.GoToDefinition;
-
 namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api
 {
     internal static class VSTypeScriptWellKnownSymbolTypes
     {
-        public const string Definition = WellKnownSymbolTypes.Definition;
+        public const string Definition = nameof(Definition);
```

Comment:
note: Thsi is kept around for binary compat (TS does reference this).  But in the past, and in the present, the value is never actually used for any purposes.  I think maybe someone wanted this to be a flexible system with more types of symbols types.  but that has never been a thing as of yet.